src/utils/constants.ts

```diff
@@ -13,4 +13,4 @@ export const EXPECTED_PERMISSIONS_BLOCK = [
 // The version of the environment in which a replay is executed. This should be bumped whenever
 // the environment changes in a way that would cause a replay to behave differently, e.g. upgrading to a newer
 // replay-orchestrator-launcher version, or changing the version of puppeteer.
-export const LOGICAL_ENVIRONMENT_VERSION = 1;
+export const LOGICAL_ENVIRONMENT_VERSION = 2;
```

Comment:
Due to https://github.com/alwaysmeticulous/meticulous-sdk/commit/c6e5dd38c84cb729ca565b668ed51e749ca08a81